src/main/java/net/dv8tion/jda/api/entities/Member.java

```diff
@@ -392,20 +393,36 @@ default ImageProxy getEffectiveAvatar()
      * <p>This is determined by the color of the highest role assigned to them that does not have the default color.
      * <br>If all roles have default color, this returns null.
      *
+     * @deprecated It's better to use {@link RoleColors#getPrimaryColor() getColors().getPrimaryColor()}.
+     *
      * @return The display Color for this Member.
      *
      * @see    #getColorRaw()
      */
     @Nullable
+    @Deprecated
     Color getColor();
 
+    /**
+     * The colors and coloring style this {@link net.dv8tion.jda.api.entities.User User}'s {@link net.dv8tion.jda.api.entities.Role Role} is displayed in.
+     *
+     * @return {@link net.dv8tion.jda.api.entities.Role.RoleColors RoleColors} associated with this user's role.
+     *
+     * @see RoleColors
```

Comment:
Please align the content of all `@` tags


```suggestion
     * @see    RoleColors
```